src/main/java/org/apache/netbeans/nbm/CreateClusterAppMojo.java

```diff
@@ -319,7 +325,24 @@ else if ( path.endsWith( ".external" ) ) // MNBMODULE-138
                                             {
                                                 outstream = new BufferedOutputStream( new FileOutputStream( fl ) );
                                                 InputStream instream = jf.getInputStream( ent );
-                                                IOUtil.copy( instream, outstream );
+                                                if ( ispack200 )
+                                                {
+                                                    Pack200.Unpacker unp = Pack200.newUnpacker();
```

Comment:
Eric, if you hit this line on JDK14+, then you get a `LinkageError` - it'd be better to catch it or prevent it and fail the build with proper "maven build exception".

Reply:
Hope this is good now